Flake8: missing whitespace after keyword

> `./punx/cache_manager.py:504:15: E275 missing whitespace after keyword`

_Originally posted by @prjemian in https://github.com/prjemian/punx/issues/216#issuecomment-1310599143_

--- a/punx/cache_manager.py
+++ b/punx/cache_manager.py
@@ -501,12 +501,12 @@ class NXDL_File_Set(object):
         if self.ref is None:
             return object.__str__(self)
 
-        return(
+        return (
             "NXDL_File_Set("
             f"last_modified={self.last_modified}"
             f", cache={self.cache}"
             f", short_sha={get_short_sha(self.sha)}"
-            f", path= {self.path}"
+            f", path={self.path}"
             ")"
         )
 

